src/engine/schema.rs

```diff
@@ -27,7 +27,7 @@ use std::sync::Arc;
 /// [`Engine`]: ../struct.Engine.html
 #[derive(Clone, Debug, PartialEq)]
 pub struct Info {
-    name: String,
+    pub(crate) name: String,
```

Comment:
Is the pub(create) `name` function to retrieve the name inadequate?

Reply:
This was a hack to be able to clone Info and "adjust" the level up to "Query" to be able to use it in `read_nodes` in the event facade. Is there a cleaner way to use Info in that situation (this is obviously a less than ideal solution).